modules/nf-core/modules/bwameth/align/main.nf

```diff
@@ -19,22 +19,19 @@ process BWAMETH_ALIGN {
     }
 
     input:
-    tuple val(meta), path(reads), path(index)
+    tuple val(meta), path(reads)
+    path index
 
     output:
     tuple val(meta), path("*.bam"), emit: bam
-    path  "*.version.txt"         , emit: version
+    path  "versions.yml"          , emit: versions
 
     script:
-    def software   = getSoftwareName(task.process)
     def prefix     = options.suffix ? "${meta.id}${options.suffix}" : "${meta.id}"
     def read_group = meta.read_group ? "-R ${meta.read_group}" : ""
     """
     INDEX=`find -L ${index} -name "*.bwameth.c2t" | sed 's/.bwameth.c2t//'`
 
-    # Modify the timestamps so that bwameth doesn't complain about building the index
-    touch -c -- *
-
```

Comment:
Not sure if we can get away with removing this. I added it after a lot of problems with bwameth.

@phue if you're reading this do you remember why it was removed in the local -> nf-core/modules switch?

Reply:
I see that thing for the first time :sweat_smile: 
Looks like you added that in June https://github.com/nf-core/methylseq/commit/f6b18b352f087e08cd265644c14486505586e517

But it doesn't seem to be on master

Reply:
Right, yeah then that'd do it 😅  Sorry, getting my timeline mixed up..

Was added into `dev` in June, but you're right - not out in a release yet: https://github.com/nf-core/methylseq/pull/217

Reply:
Added to the module: https://github.com/nf-core/modules/pull/1037

Reply:
Thanks for the merge! Pipeline updated with new version of the module.. ✅